src/Metadata/Resource/ResourceMetadataCollection.php

```diff
@@ -24,6 +24,8 @@
  */
 final class ResourceMetadataCollection extends \ArrayObject
 {
+    private const GRAPHQL_PREFIX = 'graphql_';
+    private const HTTP_PREFIX = 'http_';
```

Comment:
```suggestion
    private const GRAPHQL_PREFIX = 'g_';
    private const HTTP_PREFIX = 'h_';
```

As it's entirely internal, let's save a few bytes.